sw/device/lib/testing/test_main.h

```diff
@@ -5,14 +5,17 @@
 #ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_TEST_MAIN_H_
 #define OPENTITAN_SW_DEVICE_LIB_TESTING_TEST_MAIN_H_
 
```

Comment:
Please can you add the following doxygen header immediately after the include guard (and before any other includes):
```
/**
 * @file
 * @brief Common Definitions for whole-chip on-device tests
 */
```